config/initializers/core_extensions/devise/strategies/json_web_token.rb

```diff
@@ -5,7 +5,7 @@ module Strategies
     # strategy for login with jwt
     class JsonWebToken < Base
       def valid?
-        return false unless request.headers['Authorization'].present?
+        return false unless request.headers['HTTP_AUTHORIZATION'].present?
```

Comment:
`HTTP_AUTHORIZATION` is case-insensitive against the client header, so I would prefer to use it instead of `Authorization`.